### ISSUE TITLE
kubefirst version display log information

### ISSUE THREAD

**Issue by converge** [kubefirst/kubefirst#1221]
the `version` command displays unnecessary information for logs:

```
-----------
Follow your logs with:
   tail -f  /Users/converge/workspace/kubefirst/tmp/logs/log_1674048799.log

-----------


kubefirst-cli golang utility version: 1.11.0
```

**AC1:** it should display the version only

**Comment by echoboomer** [comment]
@jarededwards @johndietz this issue can be closed when you have a chance. :)

**Comment by jarededwards** [comment]
resolved: #1255
